docs/cow-protocol/tutorials/cow-amm-deployer.mdx

```diff
@@ -2,82 +2,43 @@
-
-The page displays the following information:
-- **Total Value**: The total value of the liquidity provided to the AMM, calculated based on the current token prices.
-- **Token Amounts**: The individual amounts of each token in the AMM liquidity pool.
-- **Price Oracle**: The selected price oracle used by the AMM.
-- **Swaps Link**: A link to view the swaps executed by the CoW AMM on the CoW Explorer.
-
-From the "Manager" page, you can perform two actions:
-1. **Stop AMM LP Position**: Clicking this button will redirect you to a confirmation page. After confirming, the CoW AMM will be stopped, and the tokens in your Safe wallet will no longer be used to provide liquidity.
-2. **Edit AMM LP Parameters**: This action will redirect you to the AMM creation form, pre-filled with the current parameters of the AMM. You can modify the parameters as needed and push the changes your existing AMM.
-
-![Manager page](/img/cowammdeployer/manager.png)
-
-## Additional Resources
-- [CoW AMM Documentation](https://cow.fi/cow-amm)
-- [CoW Grants DAO](https://grants.cow.fi/)
-- [Safe Wallet Documentation](https://help.safe.global/)
-- [CoW Swap (Safe App)](https://app.safe.global/share/safe-app?appUrl=https%3A%2F%2Fswap.cow.fi&chain=eth)
-
-## Troubleshooting and FAQs
-
-### Frequently Asked Questions
-1. **Can I create multiple CoW AMMs with the same Safe wallet?**
-   - No.
-
-2. **What happens to my tokens when I stop an AMM LP position?**
-   - When you stop an AMM LP position, your tokens in your Safe will no longer be used to provide liqudity. As the CoW AMM **IS** the Safe when there is an active CoW AMM, the liquidity never leaves your Safe wallet! You can then withdraw or use them for other purposes.
-
-3. **Can I change the token pair of an existing AMM?**
-   - No, once an AMM is created with a specific token pair, you cannot change the token pair. If you want to provide liquidity for a different token pair, you need to create a new AMM.
-
-For further assistance or questions, join the [CoW Protocol Discord](https://discord.com/invite/cowprotocol) and ask in the `#tech-talk` channel.
+
+3. The next step is to bind the approved tokens to the pool, by using the `bind()` function of the newly created pool contract. We need to do one bind per token that will be added to the pool. Note that the `denorm` parameter should always be set to 1000000000000000000 (i.e., 10^18).
+
+
+4. We can then set the swap fee; this is the fee trades will pay if they trade permissionlessly (i.e., outside of the batch), and this is done by calling the `setSwapFee()` function of the pool contract. The fee parameter needs to be specified in wei, hence, a 10% fee would be 100000000000000000 (i.e., 10^17)
+
+:::caution
+
+The fee parameter is an unsigned integer where 10^18 corresponds to 100% of fee. Note that 100% is actually not a valid value though, but one can set up to 99.99% (this needs to be verified).
```

Comment:
For full LVR protection the fee should be 100%. We need to educate the user about this or else they might mistakenly set the fee amount low and hence not be protected against LVR.